Wikipedia/Code/AppDelegate.m

```diff
@@ -81,7 +81,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     [NSUserDefaults wmf_migrateToWMFGroupUserDefaultsIfNecessary];
     [[NSUserDefaults wmf] wmf_migrateFontSizeMultiplier];
-    NSUserDefaults.wmf.shouldShowLastReadArticleOnResume = [self shouldShowLastReadArticleOnResume];
+    NSUserDefaults.wmf.shouldRestoreNavigationStackOnResume = [self shouldRestoreNavigationStackOnResume];
```

Comment:
I might just be misremembering the original ask on these dates but at this point seems like wmf_appBecomeActiveDate isn't set for this session yet so it reflects the last persisted wmf_appBecomeActiveDate from their last session. You won't get an accurate date to verify that it is launched 5 hours later since they last resigned.

Reply:
great catch, fixed!